tests/PHPStan/Analyser/AnalyserIntegrationTest.php

```diff
@@ -569,6 +570,16 @@ public function testBug6740(): void
 		$this->assertNoErrors($errors);
 	}
 
+	public function testBug1088(): void
+	{
+		$str = str_repeat('a', 99);
```

Comment:
These 3 lines should be moved into the data/*bug added file.

`assertType` is meant for use in these data/* files